### ISSUE TITLE
[BITV] 9.1.4.4/7.1 - The button to copy a log entry is not accessible when zoomed to 200%. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36961]
- [ ] Fix
- [ ] Backport
- [ ] Also ensure button is there on 320px width as well as 1280px at 400% zoom

![cabff6169b9f9f21f84523de85c61c46](https://user-images.githubusercontent.com/1315170/222229831-a2488b9a-49a0-4226-ac12-57d209951450.png)
<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-c41a0aed27-v7-n1

</details>
